terminus-ui/chip/src/chip.component.scss

```diff
@@ -5,13 +5,20 @@
 @import './../../scss/helpers/typography';
 
 
-$ts-chip-focus-background-color: color(utility);
-
 .ts-chip {
+  --chip-fontSize: 14px;
+  --chip-height: #{spacing(large)};
+  --chip-padding: #{spacing(small, 2)} #{spacing(small, 1)};
+  --chip-backgroundColor: #{color(utility, light)};
+  --chip-backgroundColor-hover: #{color(utility)};
+  --chip-borderRadius: 16px;
+  --chip-color: #{color(pure, dark)};
+  --chip-remove-size: #{spacing(large)};
+
   &:focus,
   &:hover {
     .c-chip {
-      background-color: $ts-chip-focus-background-color;
+      --chip-backgroundColor: var(--chip-backgroundColor-hover);
```

Comment:
what's the difference between setting a variable like before and this?

Reply:
The `$foo` is a Sass variable which is only available at compile time. While the `var(--foo)` format is a CSS custom property which is available at runtime.

Looking to move that direction for a couple reasons:
a) They have pretty good support at this point
b) They aren't tied to a specific library or build tool
c) They are available at runtime so we don't need to pre-generate styles for every theme we support for every component.

Sass variables are still fine for local variables that don't need to be changed. And honestly I'm still working out the details in my head so we won't be rejecting PRs using Sass vars etc. I'm just getting my feet wet here because I do think it will make overwrites/themes/extensions much more clean.